src/MainRectangle.java

```diff
@@ -2,6 +2,10 @@ public class MainRectangle {
 
     public static void main(String[] args) {
 
```

Comment:
Here are some suggestions to improve your code:

1. **Typo in 'Systom'**: There is a typo in your code. It should be `System.out.println(str);` instead of `Systom.out.println(str);`.

2. **Avoid Creating Unnecessary Objects**: Instead of `String str = new String("print me");`, use `String str = "print me";`. This avoids creating an unnecessary `String` object.

3. **Remove Unnecessary Blank Line**: There is an extra blank line after your `main` method starts. Consider removing it for cleaner code.